create_table.lua

```diff
@@ -1,21 +1,23 @@
 box.cfg{listen = 3301, memtx_memory = 10 * 1024^3}
 box.schema.user.grant('guest', 'read,write,execute', 'universe')
 
-box.sql.execute("drop table if exists warehouse;")
-box.sql.execute("create table warehouse ( \
+local sql_execute = box.sql and box.sql.execute or box.execute
```

Comment:
what's the point to keep references to outdated names like `box.sql.*`? Why not simply change them to `box.*`?

Reply:
This way we can run the benchmark on old tarantool versions to compare with them, I guess.

Reply:
As far as I can see `box.sql.execute` has been renamed to `box.execute` with this https://github.com/tarantool/tarantool/commit/9ebc41dc41dbd4bb28da46f4218851b267556611 commit for 2.2.1. 

So the question is - do we really plan to run it on 2.1? Does it make any sense today?

Reply:
It is possible, that we will be running some retrospective analysis on older versions (and they 100% are going to have schema compatibility issues) but as long as this fix is so small, I think it is a good idea to keep it here